### DISCUSSION TITLE
Is there a way open window from systemtray?

### DISCUSSION THREAD

**Discussion by GiantappMan** [tauri-apps/tauri#3872] [Q&A] [answered]
I'm guessing it has something to do with `window_builder` but I'm a newbie so looking for help here

**Comment by FabianLars** [comment, marked as answer]
For a start you can take a look at how we do it in our example app: <https://github.com/tauri-apps/tauri/blob/dev/examples/api/src-tauri/src/main.rs#L132-L137>
